cibuildwheel/macos.py

```diff
@@ -276,17 +276,20 @@ def build(options: Options, tmp_path: Path) -> None:
     )
 
     try:
-        before_all_options_identifier = python_configurations[0].identifier
-        before_all_options = options.build_options(before_all_options_identifier)
-
-        if before_all_options.before_all:
-            log.step("Running before_all...")
-            env = before_all_options.environment.as_dictionary(prev_environment=os.environ)
-            env.setdefault("MACOSX_DEPLOYMENT_TARGET", "10.9")
-            before_all_prepared = prepare_command(
-                before_all_options.before_all, project=".", package=before_all_options.package_dir
-            )
-            shell(before_all_prepared, env=env)
+        if len(python_configurations) > 0:
```

Comment:
Can we convert these blocks into early exits instead? Would avoid the extra indentation.